src/dynacli.py

```diff
@@ -49,11 +49,33 @@ class ArgProps(TypedDict):
 
     dest: str
     nargs: Optional[Union[int, str]]
-    choices: ChoicesType
+    choices: Optional[list[str]]
     type: Union[type, callable]
     action: Union[str, callable]
 
 
+def _choices_patch():
```

Comment:
This is mainly to increase the readability of the codebase. I was thinking about not provide free function with `self` as first argument, when you read the code from top to bottom. When somebody sees the `self` - the question raises wait where is the class then?
Instead wrapping it will give a taste that it is something will be used in the future - at least I think so :)